docs/api/app.md

```diff
@@ -1222,7 +1222,10 @@ This method can only be called before app is ready.
 Returns `Boolean` - Whether the application is currently running from the
 systems Application folder. Use in combination with `app.moveToApplicationsFolder()`
 
-### `app.moveToApplicationsFolder()` _macOS_
+### `app.moveToApplicationsFolder([handler])` _macOS_
+
+* `handler` Function (optional) - A handler for potential conflict in move failure.
```

Comment:
To allow cleaner API expansion this should probably be an `options` object with a single property `conflictHandler`